Source/Engine/Scripting/DotNet/CoreCLR.cpp

```diff
@@ -129,12 +129,7 @@ void CoreCLR::RegisterNativeLibrary(const char* moduleName, const char* modulePa
 
 void* CoreCLR::Allocate(int size)
 {
-#if PLATFORM_WINDOWS
-    void* ptr = CoTaskMemAlloc(size);
-#else
-    void* ptr = malloc(size);
-#endif
-
+    void* ptr = Platform::Allocate(size, 16);
 #if COMPILE_WITH_PROFILER
     Platform::OnMemoryAlloc(ptr, size);
```

Comment:
You can remove `OnMemoryAlloc`/`OnMemoryFree` calls when using `Platform::Allocate`/`Free` (it already calls profiler info)